yshi/HW1/css/hw1.css

```diff
@@ -0,0 +1,66 @@
+html {
+	border-top: 1px solid #7A7A7A;
+	width: 1024px;
+}
+body {
+	background: url("../images/page-mock.png");
+	background-size: 1024px 885px;
+	background-repeat: no-repeat;
+	font-family: Verdana;
+}
+h1 {
+	font-size: 36pt;
+	font-weight: 100;
+	padding: 11px 0px 0px 259px;
+	color: #666666;
+	margin-bottom: 25px;
+}
+h2 {
+	color: #FF6500;
+}
+img {
+    padding: 2px 0px 0px 34px;
+	-webkit-filter: contrast(1.2) brightness(0.9) saturate(1.1);
```

Comment:
intention, and if you use multiple browser support pre-fix, please include webkit os ms etc...